keras_hub/src/models/roformerV2/roformerV2_text_classifier.py

```diff
@@ -0,0 +1,122 @@
+from keras_hub.src.api_export import keras_hub_export
+from keras_hub.src.models.bert.bert_text_classifier import BertTextClassifier
+from keras_hub.src.models.roformerV2 import (
```

Comment:
just leave this like the rest of the imports and add `# noqa: E501` to the line.
https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/models/stable_diffusion_3/stable_diffusion_3_backbone.py#L7